PopPUNK/__main__.py

```diff
@@ -513,7 +560,7 @@ def main():
                      args.threads, args.use_mash, args.mash, args.overwrite, args.plot_fit, args.no_stream,
                      args.max_a_dist, args.model_dir, args.previous_clustering, args.external_clustering,
                      args.core_only, args.accessory_only, args.phandango, args.grapetree, args.info_csv,
-                     args.rapidnj, args.perplexity)
+                     args.rapidnj, args.perplexity, args.assign_lineage, args.existing_scheme, args.R, args.use_accessory)
```

Comment:
Looks like this would need to process `args.R` again - best to put that line at the top then

Reply:
Rank list only calculated once in effb8909e329c8ac57e1bc9aea09be1088f7021d.